package.json

```diff
@@ -33,14 +33,18 @@
     "dist/assets/styles/configuration.css"
   ],
   "dependencies": {
+    "@lit/context": "^1.1.1",
+    "@lit/task": "^1.0.0",
     "@xmpp/jid": "^0.13.1",
     "async": "^3.2.2",
     "decache": "^4.6.0",
     "escape-html": "^1.0.3",
     "got": "^11.8.2",
     "http-proxy": "^1.18.1",
+    "lit": "^3.1.3",
     "log-rotate": "^0.2.8",
     "openid-client": "^5.6.5",
+    "rxjs": "^7.8.1",
```

Comment:
really needed? In dependencies, not devDepedencies?

Reply:
Not needed now. I got rid of observable uses.